python/dgl/graphbolt/minibatch.py

```diff
@@ -225,3 +226,50 @@ def to_dgl_blocks(self):
                     block.edata[dgl.EID] = subgraph.reverse_edge_ids
 
         return blocks
+
+
+def exclude_edges(
+    minibatch: MiniBatch,
+    edges: Union[
+        Dict[str, Tuple[torch.Tensor, torch.Tensor]],
+        Tuple[torch.Tensor, torch.Tensor],
+    ],
+):
+    """
+    Exclude edges from the sampled subgraphs in the minibatch.
+
+    Parameters
+    ----------
+    minibatch : MiniBatch
+        The minibatch.
+    edges : Dict[str, Tuple[torch.Tensor, torch.Tensor]] or Tuple[torch.Tensor, torch.Tensor]
+        The edges to be excluded.
+    """
+    minibatch.sampled_subgraphs = [
+        subgraph.exclude_edges(edges)
+        for subgraph in minibatch.sampled_subgraphs
+    ]
+    return minibatch
+
+
+def exclude_seed_edges(minibatch: MiniBatch):
```

Comment:
Another option is to combine these 3 functions in to one:
`
exclude_edges(minibatch, remove_reverse: bool, reverse_edge_types: Dict = None)`